docs/schema.md

```diff
@@ -752,9 +754,28 @@ is required for each `credits` entry.
 Each `credits[].contact[]` entry should be a valid, fully qualified, plain-text URL
 at which the credited can be reached. Providing contacts is optional.
 
+### credits[].type[] field
+
+The `credits[].type[]` field should specify the type or role of the individual or entity
```

Comment:
The _optional_ credits[].type[] field ...

Reply:
Fixed!